tests/cases/fourslash/navigationBarItemsMultilineStringIdentifiers1.ts

```diff
@@ -7,6 +7,10 @@
 ////}
 ////declare module "MultilineMadness" {}
 ////
+////declare module "Multiline\

```

Comment:
Change looks good. Just for future if you could number each Madness so result shows exactly which one is  turned into which one would be great. Thank you.

That is would prefer this instead of current test
```ts
////declare module "Multiline\r\nMadness1" {
////}
////
////declare module "Multiline\
////Madness2" {
////}
////declare module "MultilineMadness3" {}
////
////declare module "Multiline\
////Madness4" {
////}
////
```